### PR TITLE
part 2 GA guide

### PR DESCRIPTION
**Pull request by priyankatuteja**
part 2 GA guide

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [ ] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [ ] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('https://www.arcgis.com', 'arcgis_python', 'P@ssword123')`
    - `gis = GIS(profile="your_online_profile")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal', 'arcgis_python', 'amazing_arcgis_123')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [ ] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [ ] Code refactored & split out across multiple cells, useful comments?
- [ ] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [ ] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [ ] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

**Comment by priyankatuteja** [comment]
@AtmaMani I need to clean up this notebook and re-run the steps. Will notify you as it gets ready for review

File: guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb [hunk 1]
```diff
@@ -0,0 +1,730 @@
+{
```
Comment (guneetmutreja) [inline guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb:1]: <p>..</p><p>HDFS - <strong>a</strong>&nbsp;<a href="https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#Introduction" target="_blank">Hadoop Distributed File System</a>&nbsp;directory of datasets</p><p>..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (guneetmutreja) [inline guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb:1]: <p>In the below lines the code is updated and the use of add_cloudstore() is done but the narratives also need a change accordingly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (guneetmutreja) [inline guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb:1]: <p>Please add narratives to the cell block which tells how to use add_bigdata() to add big data from a cloud store.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (guneetmutreja) [inline guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb:1]: <p>Please add narratives to the cell block which tells how to use add_bigdata() to add big data from a local machine.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (guneetmutreja) [inline guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb:1]: <p>I think there is no such item with title "FileShareFolder" in the item list below. Please recheck.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (guneetmutreja) [inline guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb:1]: <p>Two full stops : "A query of the resulting manifest returns each dataset's schema..".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

**Review by guneetmutreja — CHANGES_REQUESTED** [review]
The guide looks good overall. Just a few changes required.

**Review by guneetmutreja — CHANGES_REQUESTED** [review]
The guide looks good overall. Just a few changes required.

**Review by guneetmutreja — CHANGES_REQUESTED** [review]
The guide looks good overall. Just a few changes required.

**Comment by priyankatuteja** [comment]
will send a fresh pr along with other guides chapters

**Pull request closed without merging**